test/utils/database-url.unit.test.js

```diff
@@ -72,15 +35,72 @@ describe('utils > buildDatabaseUrl', () => {
             dbPassword: '__dbPassword__',
             dbPort: '__dbPort__',
             dbUser: '__dbUser__',
-            mongodbSrv: true,
           };
           const databaseUrl = buildDatabaseUrl(config);
 
           expect(databaseUrl).toBe(
             '__protocol__://__dbUser__:__dbPassword__@__dbHostname__:__dbPort__/__dbName__',
           );
         });
+
+        describe('when given the MongoDB SRV option', () => {
+          it('should include it when dialect is "mongodb" and ignore port', () => {
+            expect.assertions(1);
+
+            const config = {
+              dbDialect: 'mongodb',
+              dbHostname: '__dbHostname__',
+              dbName: '__dbName__',
+              dbPassword: '__dbPassword__',
+              dbPort: '__dbPort__',
+              dbUser: '__dbUser__',
+              mongodbSrv: true,
+            };
+            const databaseUrl = buildDatabaseUrl(config);
+
+            expect(databaseUrl).toBe(
+              'mongodb+srv://__dbUser__:__dbPassword__@__dbHostname__/__dbName__',
+            );
+          });
+
+          it('should ignore it otherwise', () => {
```

Comment:
This is not correct. You are not testing ONLY the srv option here, you are also testing that the conenctionUrl is correctly built, whatever the dialect. 

This comes from an upper test you relocated. It should stay as it was before `should build the URL from the configuration`